src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -221,7 +232,26 @@ def get_name(self, enable_validation: bool = False, **kwargs):
     # pylint: disable=unused-argument
     def get_ssh_key_value(self, enable_validation: bool = False, **kwargs):
         # read the original value passed by the command
-        ssh_key_value = self.raw_param.get("ssh_key_value")
+        raw_value = self.raw_param.get("ssh_key_value")
+        # try to read the property value corresponding to the parameter from the `mc` object
+        mc_property = None
```

Comment:
bad name.

Reply:
OK, will replace with a better name in another PR since before the change of this PR, there were already some variables named in this way.